crates/spirv-std/src/lib.rs

```diff
@@ -109,7 +109,14 @@ pub(crate) mod sealed;
 pub mod vector;
 
 pub use self::sampler::Sampler;
-pub use crate::macros::Image;
+/// A macro for creating SPIR-V `OpTypeImage` types and the type itself
+/// See also the re-export of this macro in the [`image`] module, which has more thorough
+/// documentation.
+///
+/// Note: The documentation below results from this being a re-export from the [`spirv-std-macros`](`crate::macros`) crate.
+/// It can be safely ignored
+///
+pub use crate::image::Image;
```

Comment:
Why do a re-re-export instead of a re-export?

Reply:
It was a re-reexport in an attempt to get the docs properly copied over

Obviously that doesn't work, so it can just be a normal export again.